src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -353,7 +353,7 @@
             "operations": [
                 {
                     "method": "GET",
-                    "summary": "Get partition state for current node",
+                    "summary": "Get partition state. (IMPORTANT: this API returns an identical result on all nodes) ",
```

Comment:
nit: I think this can be simplified to something like .. "Get low level debug information of all replicas of a given partition" or some such.

Reply:
Makes sense. How about this? Just wanted to make it explicit we can run it on any node.
```
Get low level debug information (on any node) of all replicas of a given partition
```

Reply:
I felt "on any node" part is an implementation detail but I'm not too strong about, ok with the wording you suggested.